data/json/snippets/e_logs.json

```diff
@@ -123,6 +123,86 @@
+      },
+      {
+        "id": "local_files_simple_36",
+        "text": "This security tape captures the view of a rural gas station and the deserted, tree-fringed roadway beyond.  The space beside one of the gas pumps is soon filled as a large 4x4 motors into the frame, towing a bouncing and rocking animal carriage behind it.  As the truck pulls up and a burly, bearded man clambers out, insistent whinnies and heavy bangs emanate from the trailer’s interior.  Brows nit in concern, the driver potters with the pump as the trailer trembles and shakes with the struggles of its displeased captive, the neighs growing ever harsher and verging on a distorted scream by the time the man’s finished tending to his vehicle.  After he finishes gassing up, the motorist replaces the nozzle and approaches the trailer, rapping on its side.\n\n\"Blacky,\" he calls out, \"the hell is wrong with you in there?\"\n\nA crushing blow bulges the trailer’s shell, causing it to sway violently as rivets burst and the man staggers back.  With successive blows shattering the trailer’s side panels, the head of a horse eventually forces its way through a widening crack, shards of jagged metal stripping the flesh from its skull as it struggles.  Scrambling to the door of his 4x4, the driver flings himself into the vehicle as the beast crashes free, sniffing the air for a few moments before rearing up and bringing its front hooves down on the truck's windshield.  Cowering out of sight, the man’s sobbing pleas and screams are audible as the heavy utility vehicle is pummeled into a crumpled wreck, the heap of mobile sinew that used to be his steed struggling to push its snapping snout through the shattered window.\n\nTearing the driver’s side door free with a screech of ripping metal, the animal tosses its head and lets loose a malformed shriek before it plunges into the peeled-open cabin.  Muscles rippling and thrashing, the horse’s metal-flayed bulk obscures much of the scene, though the man’s rising screams, the nauseating sounds of bones being torn from their homes, and the sprays of viscera pulsing through the vehicle’s open gaps attest well enough to the quadruped’s brutal work."
```

Comment:
The injury to the horse sounds fatal, yet it does not appear to be zombified just yet. This might be a minor lore inconsistency here.

I'd suggest rewording to either make it clear it is zombified or make the injuries a bit less severe.

Reply:
Hey, thanks for the review. I pushed across the changes, and you think this slight tweak’s good enough to establish the horse as being zombified?

Reply:
Thanks github for making the diff almost impossible to read. Yeah, the addition of "undead" make it clear enough.